src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    Random random = new Random();
```

Comment:
Consider making the 'random' variable private to adhere to encapsulation principles. Class variables should generally be private and accessed through methods if needed.